packages/tupaia-web/src/layout/Map/MapLegend/MapLegend.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { ExpandableMapLegend } from './ExpandableMapLegend';
+import { FixedMapLegend } from './FixedMapLegend';
+
+export const MapLegend = () => {
+  return (
+    <>
+      <ExpandableMapLegend />
```

Comment:
Even though the names make sense, it took me a few seconds to realise what each of these was. What do you think about just calling them DesktopMapLegend and MobileMapLegend?